docs/tutorials/dronecan_flight_controller.rst

```diff
@@ -93,25 +102,35 @@ You can change the ESC Index of a module through the Control Center using *ESC I
 
 Arming
 ----------
-Vertiq modules can use :ref:`Advanced Arming <manual_advanced_arming>` with DroneCAN to improve safety and allow for configurable disarming behaviors. To see if your module 
-supports Advanced Arming, refer to the :ref:`Module Support section under Advanced Arming <arming_module_support>`. When using arming, modules must arm before they can spin.
-The default settings for Vertiq modules generally allow them to arm when they receive throttle commands between 0% and 7.5%, and by default both PX4 and ArduPilot send 0% commands when 
-properly configured and disarmed. So the default settings on Vertiq modules should allow them to arm immediately when they are connected to a properly configured flight controller on a DroneCAN bus. 
 
-**Because of this, there is no need to change any arming parmeters to complete a basic integration with a PX4 or ArduPilot flight controller.** If you wish to take advantage of these arming
-features for more complex integrations, refer to the :ref:`Advanced Arming <manual_advanced_arming>` section for more details.
+If you intend to leverage Vertiq's :ref:`Advanced Arming <manual_advanced_arming>`, there are two main options for controlling your module's armed state with DroneCAN. 
+First, and recommended, is via DroneCAN's :ref:`ArmingStatus message <dronecan_arming_and_bypass>`, and second via throttle commands. When using the arming feature, modules must arm before they can spin.
+
+Arming with ArmingStatus (Recommended)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 If your module is set to :ref:`use ArmingStatus to drive module arming <dronecan_arming_and_bypass>`, the module will arm on the status of your flight controller's reported ArmingStatus. 
 Configuring your flight controller to output the ArmingStatus message is covered in the specific flight controller sections below.
 
 .. note::
-	In order to control your module properly while arming with ArmingStatus messages, you must properly set the :ref:`manual_arming_throttle_source` parameter. This parameter should 
-	be set to the protocol being used to send the module throttle commands.
+	
+    When arming with the ArmingStatus message, you should disable your module's ability to arm with throttle. To do so, simply set :ref:`arm on throttle <arming_throttle_regions>` to 
+    *Do Not Arm on Throttle*. If you do not disable arm on throttle, then your module can end up in a loop of arming and disarming which may lead to unexpected and potentially 
+    dangerous behaviors. Ensuring that your module only arms and disarms based on your flight controller's armed state increases safety for you and your modules.
```

Comment:
By the way, we don't have Use Arming Status turned on by default in the new firmware right? I don't think so, just double checking so a bunch of people don't suddenly start hitting weird loops like this

Reply:
correct, it is off by default